modules/installation-special-config-storage.adoc

```diff
@@ -44,7 +44,7 @@ This feature:
 * Is supported on {op-system-first} systems only
 * Sets up disk encryption during the manifest installation phase, encrypting all data written to disk, from first boot forward
 * Requires no user intervention for providing passphrases
-* Uses AES-256-XTS encryption, or AES-256-CBC if FIPS mode is enabled
+// * Uses AES-256-XTS encryption, or AES-256-CBC if FIPS mode is enabled
```

Comment:
Should we leave the _Uses AES-256-XTS encryption_ in?

Reply:
Yes. You are correct. I will update that. Nice catch.